components/TTSPlayerModal.vue

```diff
@@ -210,8 +223,11 @@ const scrollIndicatorClasses = [
   'to-transparent',
   'pointer-events-none',
 ]
-
+const config = useRuntimeConfig()
+const avatarURL = config.public.baseURL
```

Comment:
We shouldn't need this, please refer to comment below about dynamic import static asset